c++/src/ColumnWriter.cc

```diff
@@ -1222,6 +1269,18 @@ namespace orc {
     }
   }
 
+  void StringColumnWriter::finishStreams() {
+    ColumnWriter::finishStreams();
+    if (useDictionary) {
+      dictDataEncoder->finishEncode();
+      dictLengthEncoder->finishEncode();
+      dictStream->finishStream();
```

Comment:
Why calling this on dictionary stream? It is shared by all row groups so alignment looks unnecessary to me.